_posts/2021-04-27-rust-dev-workflow.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "A New Development Workflow for Arrow's Rust Implementation"
+date: "2021-04-27 00:00:00"
+author: ruanpa
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow Rust community is excited to announce that its migration to a new development workflow is now complete! If you're considering Rust as a language for working with columnar data, read on and see how your use case might benefit from our new and improved project setup.
+
+In recent months, members of the community have been working closely with Arrow's [Project Management Committee][1] and other contributors to expand the set of available workflows for Arrow implementations. The goal was to define a new development process that ultimately:
+- Enables a faster release cadence that adheres to [SemVer][15] where appropriate
+- Encourages maximum participation from the wider community with unified tooling
+- Ensures that we continue to uphold the tenets of [The Apache Way][2]
+
+If you're just here for the highlights, the major outcomes of these discussions are as follows:
+- The Rust projects have moved to separate repositories, outside the main Arrow [monorepo][9]
+	- [arrow-rs][7] for the core Arrow implementation in Rust
+	- [arrow-datafusion][8] for DataFusion and Ballista (more on these projects below!)
+- The Rust community will use GitHub Issues for tracking feature development and issues, replacing the Jira instance maintained by the Apache Software Foundation (ASF)
+
+But why, as a community, have we decided to change our processes? Let's take a slightly more in-depth look at the Rust implementation's needs.
```

Comment:
Possibly worth mentioning that this is a shift back towards the process used by Ballista?

Adds the context of "going back to a thing that worked" rather than "let's try something new".